plugins/2.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: "2"
+spec:
+  version: "v0.0.0"
+  shortDescription: dumb file transfer to your labelled pods
+  homepage: https://steady.supply/git/kubectl-2/
+  description: |
+    it all starts with a configuration file
+
+    spec:
+      - namespace: xyz
+        label: sync=me,src=py
+        path: src:/app
+
+    which specifes a namespace to work in, a label filter used to select pods
+    and, of course, the local and remote paths involved.
+
+    the local path is watched, and when a file changes, that file is streamed
+    to all the pods matching the label specification.
+  platforms:
+    - selector:
+        matchLabels:
+          os: darwin
+      uri: https://steady.supply/git/kubectl-2/release/v0.0.0/kubectl-2-v0.0.0-darwin-amd64.tar.gz
```

Comment:
Would you consider using github + github releases?
In that case we have tooling to automate releases, as well as we track download stats of github release assets.

Reply:
I'd rather self-host. Release automation is a few lines of shell `:)`